pinot-query-runtime/src/test/resources/queries/JsonType.json

```diff
@@ -0,0 +1,41 @@
+{
+  "json_data": {
+    "tables": {
+      "jsonTbl": {
+        "schema": [
+          {"name": "jsonCol", "type": "JSON"},
+          {"name": "stringCol", "type": "STRING"}
+        ],
+        "inputs": [
+          ["{\"key1\":\"val1\",\"key2\":\"val2\"}", "str1"],
+          ["{\"key11\":\"val11\",\"key22\":\"val22\"}", "str22"],
+          ["{\"key111\":\"val111\",\"key222\":{\"key222_a\":\"val222_a\"}}", "str33"]
+        ]
+      }
+    },
+    "queries": [
+      {
+        "sql": "SELECT jsonCol FROM {jsonTbl}",
+        "outputs": [
+          ["{\"key1\":\"val1\",\"key2\":\"val2\"}"],
+          ["{\"key11\":\"val11\",\"key22\":\"val22\"}"],
+          ["{\"key111\":\"val111\",\"key222\":{\"key222_a\":\"val222_a\"}}"]
+        ]
+      },
+      {
+        "sql": "SELECT jsonCol, stringCol FROM {jsonTbl}",
+        "outputs": [
+          ["{\"key1\":\"val1\",\"key2\":\"val2\"}", "str1"],
+          ["{\"key11\":\"val11\",\"key22\":\"val22\"}", "str22"],
+          ["{\"key111\":\"val111\",\"key222\":{\"key222_a\":\"val222_a\"}}", "str33"]
+        ]
+      },
+      {
+        "sql": "SELECT jsonCol, stringCol FROM {jsonTbl} where stringCol='str33'",
+        "outputs": [
+          ["{\"key111\":\"val111\",\"key222\":{\"key222_a\":\"val222_a\"}}", "str33"]
```

Comment:
can you also add 
- `SELECT stringCol FROM {jsonTbl} where jsonCol = '<escaped json str>'` to check the equal comparison?
- `SELECT * `

Reply:
I added more tests. 

I skipped the `select * ` because I am not sure how to skip the `ts` column. The other one selecting both `json` and `string` columns is already there.